src/front/app/__test__/Input.test.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { shallow, render, mount } from 'enzyme';
+import Input from '../components/globals/Input';
+
+describe('(Component) Input', () => {
+  it('Should render', () => {
+    const wrapper = mount(
+      <Input name="random" />
+    );
+    expect(wrapper.find(Input)).toHaveLength(1);
+  });
+
+  it('Should render its children', () => {
+    const wrapper = shallow(
+      <Input name="random">
+        <option value="value">Text</option>
+      </Input>
+    );
+
+    expect(wrapper.contains(<option value="value">Text</option>)).toBe(true);
```

Comment:
Why would an input-text ever render it's children? `input` is a self-enclosed component. This only applies for Select no?

Reply:
it shouldn't, this only applies to **select** type inputs. And will (when changed) work for **textarea** types too.